adm_program/modules/userrelations/relationtypes.php

```diff
@@ -79,6 +82,18 @@
     $relRow2['urt_name_female'] = $relRow2['urt_name_female_inverse'];
     $relationtype2->setArray($relRow2);
 
+    if((bool) $relRow['urt_edit_user'])
```

Comment:
But the control structure examples are like this and have the white spaces.
Also Ximex is always updating like this.